client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -184,19 +196,27 @@ class CancelPurchase extends Component {
 		}
 
 		const { purchase, isJetpackPurchase } = this.props;
+		const locale = getCurrentLocaleSlug();
```

Comment:
`getCurrentLocaleSlug()` is a Redux selector and requires passing in the state, so this will always return `null`. You need to call this in the `mapStateToProps` function at the bottom of this file.

Reply:
`getCurrentLocaleSlug()` has been moved to the mapStateToProps, not sure why it returned the language to me